server/server.js

```diff
@@ -1,20 +1,83 @@
 const express = require("express");
 const cors = require("cors");
+const bcrypt = require("bcrypt");
+const { check } = require("express-validator");
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
+const authRouter = express.Router();
 
 const app = express();
 const port = 8080;
 app.use(express.json());
 app.use(cors());
+app.use("/auth", authRouter);

```

Comment:
tega `authRouter` servica oz paketa ne štekam glih najboljš.. kaj dela točno @Luka85 ?

Reply:
Ta middleware sem uporabil, da imam kot default /auth routo, da ne vpisujem vedno npr.` /auth/signup` ali `auth/login`, ampak samo `/login` oz  `/signup`, zato pa potem ne uporabljam `app.get` oz `app.post`, ampak imam vedno `authRouter.get` , `authRouter.post`, zato da doda ta ` /auth`. V tem primeru, ko nimam prov veliko teh routov bi lajo vedno pač vpisoval /auth/login oz auth/signup. Kaj misliš?

Reply:
po želji.. samo zanimal me je kaj dela, da ni nek black box magic 👍